### PR TITLE
Fix false positives in whitespace rules when there is an extra semicolon.

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://github.com/stylelint/stylelint/issues/1282
/cc @davidtheclark @jeddy3

### CODE REVIEW

File: src/rules/declaration-block-semicolon-newline-after/__tests__/index.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ testRule(rule, {
   accept: [ {
     code: "a { color: pink;\n}",
   }, {
+    code: "a { color: pink;;\n}",
```
Comment (davidtheclark) [inline src/rules/declaration-block-semicolon-newline-after/__tests__/index.js]: What if there is whitespace space between the semicolons, e.g.

```css
a { color: pink; ; }
```

```css
a {
  color: pink;
  ;
}
```

**Comment by jeddy3** [comment]
@evilebottnawi Can you add a test for space between semicolons. Then I think we can merge this.

**Comment by evilebottnawi** [comment]
@jeddy3 add spaces fires false positive, need more investigation for this.

**Comment by ntwb** [comment]
Per https://github.com/stylelint/stylelint/issues/1282#issuecomment-222321988, closing

**Pull request closed without merging**